omod/src/main/webapp/pages/labtrackingOrderReadOnly.gsp

```diff
@@ -1,99 +1,3 @@
-<div class="panel panel-default" ng-if="hasSpecimenCollection(order)">
```

Comment:
Is this just cleanup of something else?

Reply:
Yes, just cleanup. We moved all the specimen details to the read only panel above.